crates/cards/tests/basic.rs

```diff
@@ -0,0 +1,88 @@
+#![allow(dead_code, clippy::unwrap_used, clippy::panic_in_result_fn)]
+
+use cards::{CardExpiration, CardExpirationMonth, CardExpirationYear, CardSecurityCode};
+use common_utils::date_time;
+use masking::PeekInterface;
+
+#[test]
+fn test_card_security_code() {
+    // no panic
+    let valid_card_security_code = CardSecurityCode::try_from(1234).unwrap();
+
+    // will panic on unwrap
+    let invalid_card_security_code = CardSecurityCode::try_from(12);
```

Comment:
```suggestion
    let invalid_card_security_code = CardSecurityCode::try_from(12);
    let invalid_card_security_code_deserialization = serde_json::from_str::<CardSecurityCode>("12"); // Check if this is an error
```

Could you also add tests like this (for each of the newtypes) to ensure deserialization does not allow invalid values?